package.json

```diff
@@ -41,12 +42,16 @@
     "cov": "npm run cov:clean && npm run cov:generate",
     "cov:clean": "rimraf ./coverage/",
     "cov:generate": "babel-node node_modules/.bin/isparta cover --report text --report html test/index.js",
-    "prepublish": "npm run build && npm test",
+    "prepublish": "npm run build && npm run check && npm test",
     "test": "npm run lint && babel-node test/index.js",
     "clean": "rimraf dist/* && mkdir dist || true",
     "uglify": "uglifyjs dist/stampit.js -m -c warnings=false -o dist/stampit.min.js",
     "lint": "eslint src && eslint test",
-    "build": "npm run clean && babel src --out-dir dist && npm run uglify"
+    "build": "npm run clean && babel src --out-dir dist && npm run uglify",
+    "deps": "npm run deps:missing && npm run deps:extra",
+    "deps:missing": "dependency-check package.json",
+    "deps:extra": "dependency-check package.json --extra --no-dev --ignore",
+    "check": "npm run deps && npm outdated --depth 0"
```

Comment:
What is the purpose of running `npm outdated` here. It won't prevent publishing afaik, looks to be just some extra info for the developer.